org.freedesktop.Sdk.Extension.rust-nightly.yml

```diff
@@ -31,15 +32,25 @@ modules:
       - type: archive
         only-arches:
           - x86_64
+        dest: rust
         url: https://static.rust-lang.org/dist/2021-05-14/rust-nightly-x86_64-unknown-linux-gnu.tar.xz
         sha256: b6fdc503b8ec59d4d5cca2f93c1641961907b8c3b6ff64b2b121a6f84380e298
         x-checker-data:
           type: rust
           package: rust
           channel: nightly
           target: x86_64-unknown-linux-gnu
+      - type: archive
+        dest: rust-src
+        url: https://static.rust-lang.org/dist/2021-05-14/rust-src-nightly.tar.xz
+        sha256: 0a17d813d5df512b222941e2363b5a04a5f5e34c7bacf5bdda4d30497a350fc0
+        x-checker-data:
```

Comment:
You need to specify that you want nightly rust-src

Reply:
Where do you want this to be specified? As part of  the "dest" name? (rust-src-nightly)?

Reply:
As part of the x-checker, see other sources for an example